diff --git a/default.d.ts b/default.d.ts
index bedfdce03cdcfdb145bd23e4465fd961e47bed52..c8b1c742970ed6ed3e3f76d755222fcf01152572 100644
--- a/default.d.ts
+++ b/default.d.ts
@@ -1 +1 @@
-export * from '.prisma/client/default'
+export * from './.prisma/client/default'
diff --git a/default.js b/default.js
index 3c2dafb5b39c0acd26fb83126a6d6c6cbe0ce18b..8111f0a30a6c6232cbcb723a8d1c3026b6b645cb 100644
--- a/default.js
+++ b/default.js
@@ -1,3 +1,3 @@
 module.exports = {
-  ...require('.prisma/client/default'),
+  ...require('./.prisma/client/default'),
 }
diff --git a/edge.d.ts b/edge.d.ts
index b8d190e2e68ba08b6efbaa29030c59454f5ffc4b..aaae3c669b80f13a9339ffae1f28e1927d74c665 100644
--- a/edge.d.ts
+++ b/edge.d.ts
@@ -1 +1 @@
-export * from '.prisma/client/edge'
+export * from './.prisma/client/edge'
diff --git a/edge.js b/edge.js
index c4925e8246fdb4b7d9271665f38f55cfe7245743..c9f11630b6c0094cad07cd7df7c3163364ac3325 100644
--- a/edge.js
+++ b/edge.js
@@ -1,4 +1,4 @@
 module.exports = {
   // https://github.com/prisma/prisma/pull/12907
-  ...require('.prisma/client/edge'),
+  ...require('./.prisma/client/edge'),
 }
diff --git a/index-browser.js b/index-browser.js
index 3ea8d77d8b781f96e1926aec8dcd0c75f52d2612..3d585bbf719a8de8778f498975a302876b2977c5 100644
--- a/index-browser.js
+++ b/index-browser.js
@@ -1,3 +1,3 @@
-const prisma = require('.prisma/client/index-browser')
+const prisma = require('./.prisma/client/index-browser')
 
 module.exports = prisma
diff --git a/index.d.ts b/index.d.ts
index bedfdce03cdcfdb145bd23e4465fd961e47bed52..c8b1c742970ed6ed3e3f76d755222fcf01152572 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -1 +1 @@
-export * from '.prisma/client/default'
+export * from './.prisma/client/default'
diff --git a/index.js b/index.js
index 1be37ebf15bceb2ac01f8d81ae698e31d424fdda..337a21f7faa3954f2ebf2ed12133c37fe45b7461 100644
--- a/index.js
+++ b/index.js
@@ -1,4 +1,4 @@
 module.exports = {
   // https://github.com/prisma/prisma/pull/12907
-  ...require('.prisma/client/default'),
+  ...require('./.prisma/client/default'),
 }
diff --git a/react-native.d.ts b/react-native.d.ts
index bfcd70688308b460dd10b19066101ca06cf89596..5bfb35c7e61fe51f13122e174acbbf5df622cdc2 100644
--- a/react-native.d.ts
+++ b/react-native.d.ts
@@ -1 +1 @@
-export * from '.prisma/client/react-native'
+export * from './.prisma/client/react-native'
diff --git a/react-native.js b/react-native.js
index 12b76d33802f6f7d04636ca578c569ff6b343939..1592f6eafa8c95001b3084b2ecc22da1c88c9796 100644
--- a/react-native.js
+++ b/react-native.js
@@ -1,3 +1,3 @@
 module.exports = {
-  ...require('.prisma/client/react-native'),
+  ...require('./.prisma/client/react-native'),
 }
diff --git a/sql.d.ts b/sql.d.ts
index ff2b18fd3fe1e453f8ec6f77e61f43dda53e6a76..40476a55d81a5a47d0830c61103e0b14acc23865 100644
--- a/sql.d.ts
+++ b/sql.d.ts
@@ -1 +1 @@
-export * from '.prisma/client/sql'
+export * from './.prisma/client/sql'
diff --git a/sql.js b/sql.js
index 6d54621ba062ab0d666cab28a6993e4ede07b75c..dfefb782bc413586818f18b2438dbcf38c2f0cb2 100644
--- a/sql.js
+++ b/sql.js
@@ -1,4 +1,4 @@
 'use strict'
 module.exports = {
-  ...require('.prisma/client/sql'),
+  ...require('./.prisma/client/sql'),
 }
diff --git a/sql.mjs b/sql.mjs
index 9349dbf5088dfe59d42331e15ffd501166911b61..166a8f9c707f3a890756a06700612acc5817d9e7 100644
--- a/sql.mjs
+++ b/sql.mjs
@@ -1 +1 @@
-export * from '../../.prisma/client/sql/index.mjs'
+export * from './.prisma/client/sql/index.mjs'
diff --git a/wasm.d.ts b/wasm.d.ts
index 1a478968db2e17d42ac89de52aed9c49e054a251..4164a529ed476007392881785b0f756d0e0aa448 100644
--- a/wasm.d.ts
+++ b/wasm.d.ts
@@ -1 +1 @@
-export * from '.prisma/client/wasm'
+export * from './.prisma/client/wasm'
diff --git a/wasm.js b/wasm.js
index a63271aa18e6c5fbb4acd2619661443e06c14057..0f8ae2dcbad8634eed531a07fa35a7d81b5053be 100644
--- a/wasm.js
+++ b/wasm.js
@@ -1,4 +1,4 @@
 module.exports = {
   // https://github.com/prisma/prisma/pull/12907
-  ...require('.prisma/client/wasm'),
+  ...require('./.prisma/client/wasm'),
 }
